samples/ApptentiveSample/ViewController.cs

```diff
@@ -20,7 +20,7 @@ public override void ViewDidLoad()
 
             EngageButton.TouchUpInside += delegate {
                 var eventName = EventNameTextField.Text;
-                Apptentive.Shared.Engage(eventName, this);
+                Apptentive.Shared.Engage(eventName, this, (engaged) => Console.WriteLine("Event engaged: " + engaged) );
```

Comment:
Is this the only place we needed to support async callbacks?

Reply:
We also have it here:
https://github.com/apptentive/apptentive-xamarin-ios/blob/fd755f2b1d1f4891ce184a4456ab9cabcb8d984c/samples/ApptentiveSample/ViewController.cs#L27
https://github.com/apptentive/apptentive-xamarin-ios/blob/fd755f2b1d1f4891ce184a4456ab9cabcb8d984c/samples/ApptentiveSample/ViewController.cs#L32